.github/workflows/shared-release-pull-request-java.yml

```diff
@@ -432,7 +432,7 @@ jobs:
 
     steps:
       - name: Merge production into main
-        uses: devmasx/merge-branch@8b86512c768bec19827894f0a39a776809668189
```

Comment:
can this be v.1.4.0 or @master
https://github.com/devmasx/merge-branch/releases

Reply:
Perhaps @master however more likely to break. 1.4.0 is older than the current commit. Doesn't look like they're tagging it anymore.

Reply:
These are the changes incorported by tihs PR as it stands: https://github.com/devmasx/merge-branch/compare/8b86512c768bec19827894f0a39a776809668189...6ec8363d74aad4f1615d1234ae1908b4185c4313